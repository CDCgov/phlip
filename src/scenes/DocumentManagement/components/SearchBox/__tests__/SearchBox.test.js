import React from 'react'
import { shallow } from 'enzyme'
import { SearchBox } from '../index'

const props = {
    searchTerms: {
        docNameSearchValue: 'doc1',
        uploadedBySearchValue: 'Tim',
        uploadedDateSearchValue: '1/1/2000',
        projectSearchValue: 'Project 1',
        jurisdictionSearchValue: 'Ohio (state)',
    }
}

describe('DocumentManagement - SearchBox', () => {
  test('should render correctly', () => {
    expect(shallow(<SearchBox {...props} />)).toMatchSnapshot()
  })
})