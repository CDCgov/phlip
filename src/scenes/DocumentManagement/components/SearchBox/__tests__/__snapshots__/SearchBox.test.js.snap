// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentManagement - SearchBox should render correctly 1`] = `
<Manager>
  <Reference>
    <Component />
  </Reference>
  <Popper
    eventsEnabled={false}
    placement="bottom-start"
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  >
    <Component />
  </Popper>
</Manager>
`;
